app/data_model/list_store.py

```diff
@@ -174,3 +174,7 @@ def deserialise(cls, serialised: list):
             return cls()
 
         return cls(existing_items=serialised)
+
+    def get_first_item_from_list(self, list_name):
```

Comment:
This could be made an attribute of the ListModel, so that the `first` id selector always returns the element at index 0 of it's `items` attribute.

You wouldn't need to change rules.py then